apps/console/src/public/deployment.config.json

```diff
@@ -504,14 +504,38 @@
                     ]
                 }
             },
-            "approvals": {
+            "approvalWorkflows": {
                 "disabledFeatures": [],
                 "enabled": true,
                 "featureFlags": [],
                 "scopes": {
+                    "create": [
+                        "internal_workflow_association_create",
+                        "internal_workflow_create"
+                    ],
+                    "delete": [
+                        "internal_workflow_association_delete",
+                        "internal_workflow_delete"
+                    ],
                     "feature": [
                         "console:workflows"
                     ],
+                    "read": [
+                        "internal_workflow_association_view",
+                        "internal_workflow_view"
+                    ],
+                    "update": [
+                        "internal_workflow_association_update",
+                        "internal_workflow_update"
+                    ]
+                }
+            },
+            "approvals": {
+                "disabledFeatures": [],
+                "enabled": true,
+                "featureFlags": [],
+                "scopes": {
+                    "feature": ["console:users"],
```

Comment:
Ideally no console feature scope is required. Need to check how to accomplish that. Setting this scope temporarily.